perf/reader_test.go

```diff
@@ -279,7 +279,7 @@ func TestPerfReaderClose(t *testing.T) {
 }
 
 func TestCreatePerfEvent(t *testing.T) {
-	fd, err := createPerfEvent(0, 1)
+	fd, err := createPerfEvent(0, 1, 0)
```

Comment:
I would probably pass `ReaderOptions` here, this call site is quickly becoming incrutable. :)